src/pages/events/release-notes.md

```diff
@@ -12,6 +12,24 @@ These release notes describe the latest version of Adobe I/O Events for Adobe Co
 
 See [Update Adobe I/O Events for Adobe Commerce](installation.md#update-adobe-io-events-for-adobe-commerce) for upgrade instructions.
 
+## Version 1.8.0
+
+### Release date
+
+July 31, 2024
+
+### Enhancements
+
+* Added synchronize events button to eventing general configuration. <!--- CEXT-3337 -->
+
+* Added create event provider button to eventing general configuration. <!--- CEXT-3336 -->
```

Comment:
@keharper 
This item is added to the https://github.com/magento-commerce/module-adobe-io-events-client module but it seems we don't have separate release notes for module-adobe-io-events-client so I added it here.